controller/controller-board/bower.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "controller-board",
+  "description": "controller interface",
+  "homepage": "https://github.com/pinterest/rocksplicator",
+  "license": "MIT",
```

Comment:
put apache as well?

Reply:
The ```license``` field only allow 1 parameter otherwise it won't compile. I think it should be okay like this because this is a subproject licensed under MIT